Frontend_Web_Development_React_Angular_Vue/React/Week 2/Components.md

```diff
@@ -22,7 +22,7 @@ const App = () => {
 ReactDOM.render(<App />, document.getElementById("root"));
 ```
 This is a simplest React Application anyone can write, let's understand it line by line:
-- The first thing we do here, is to make our own component, `App`. React is all about making components. And then taking those components and making more components out of those.
+- The first thing we do here, is to make our own component, `App`. React is all about making components. And then taking those components to reuse and modularize in our application.
```

Comment:
And then taking those components to implement reusability and modularity in our application.